src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -862,4 +862,19 @@ public static void warnOnNonIlluminaReadGroups(final SAMFileHeader readsHeader,
             logger.warn("This tool has only been well tested on ILLUMINA-based sequencing data. For other data use at your own risk.");
         }
     }
+
+    /**
+     * Simple wrapper for sticking elements of a int[] array into a List<Integer>
+     * @param ar - the array whose elements should be listified
+     * @return - a List<Integer> where each element has the same value as the corresponding index in @ar
+     */
+    public static List<Integer> listFromPrimitives(final int[] ar) {
```

Comment:
done. i delegate to Ints.asList now but leave this behind so that we dont end up with the same thing again.